Listenbrainz/UI/Components/SettingsView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SettingsView.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 16/06/23.
+//
+
+import SwiftUI
+
+
+struct SettingsView: View {
+  @AppStorage("isDarkMode") private var isDarkMode = false
+
+  @State private var userToken:String = ""
+  @Environment(\.presentationMode) var presentationMode
```

Comment:
Have a look at the dismiss Environment. presentationMode has been deprecated so we can switch

Reply:
done